src/ORM/DataQuery.php

```diff
@@ -453,6 +453,41 @@ public function count()
         return $this->getFinalisedQuery()->count("DISTINCT {$quotedColumn}");
     }
 
+    /**
+     * Return whether this dataquery will have records. This will use `EXISTS` statements in SQL which are more
+     * performant - especially when used in combination with indexed columns (that you're filtering on)
+     *
+     * @return bool
+     */
+    public function exists()
```

Comment:
```suggestion
    public function exists(): bool
```

May as well since it's a new API